dev-libs/processor-trace/processor-trace-2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Intel(R) Processor Trace decoder library"
+HOMEPAGE="https://github.com/01org/processor-trace"
+SRC_URI="https://github.com/01org/processor-trace/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT=0
```

Comment:
Given that the SONAME has changed, it'd be useful to make use of subslots here, e.g.

    SLOT="0/2"

to indicate that this version installs `.so.2` and have packages automatically rebuilt when they linked against `.so.1`.

Reply:
Fixed.